types/views/views.go

```diff
@@ -275,6 +275,19 @@ func SliceContains[T comparable](v Slice[T], e T) bool {
 	return false
 }
 
+// SliceEqual is like the standard library's slice.Equal, but for two views.
```

Comment:
slices?

Reply:
whoops. I had it right in the commit message.